tests/13-ieee802154/js/security.js

```diff
@@ -0,0 +1,26 @@
+TIMEOUT(50000);
+
+var failed = false;
+var done = 0;
+
+while(done < sim.getMotes().length) {
+    YIELD();
+
+    log.log(time + " " + "node-" + id + " "+ msg + "\n");
+
+    if(msg.contains("=check-me=") == false) {
+        continue;
+    }
+
+    if(msg.contains("FAILED")) {
+        failed = true;
```

Comment:
Run `log.testFailed()` directly here, no point to continue running a test that will fail.